### PR TITLE
[Glimmer2] Port `{{loc}}` helper test to `ember-glimmer`

### PR DESCRIPTION
**Pull request by code0100fun**
Also implements helper in Glimmer

### CODE REVIEW

File: packages/ember-glimmer/tests/integration/helpers/loc-test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Ember.STRINGS = {
+      '_Howdy Friend': 'Hallo Freund'
+    };
+  }
+
+  teardown() {
+    Ember.STRINGS = this.oldString;
+  }
+
+  ['@test it lets the original value through by default']() {
```
Comment (chadhietala) [inline packages/ember-glimmer/tests/integration/helpers/loc-test.js]: These tests should go through the "The I-N-U-R cycle" mentioned in #13127.

Comment (code0100fun) [inline packages/ember-glimmer/tests/integration/helpers/loc-test.js]: Woops, right. Will fix that, thanks.

Comment (code0100fun) [inline packages/ember-glimmer/tests/integration/helpers/loc-test.js]: Seems like this only needs to check initial render and no-op rerender, not update and reset. Does that seem correct?

**Comment by mixonic** [comment]
It seems OK to punt on, but this should likely be testing the bound behaviors as well, which do exist: https://ember-twiddle.com/0ec0748d515ec3b87414?openFiles=application.controller.js%2C It seems an oversight that there are no tests for those cases.

**Comment by code0100fun** [comment]
@mixonic, I'll work on adding some tests for the bound params.

**Comment by chancancode** [comment]
Thanks! @code0100fun! Please send a follow up PR for @mixonic's request :smile:

**Pull request merged**